src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -390,7 +390,7 @@ def fail(self, e):
         assert self._state in (self.INITIALIZED, self.RUNNING)
         logger.exception('_Promise failed')
         self._exception = e
-        self._value = f'_exception: {e}'
+        self._value = '_exception: {}'.format(e)
```

Comment:
This was fixed a few days ago. no need for this change anymore.

Reply:
Thanks for informing. I will remove it.